crates/utils/src/config.rs

```diff
@@ -81,6 +81,18 @@ pub struct ClusterConfig {
     /// Get xline server name
     #[getset(get = "pub")]
     name: String,
+    /// Xline server peer listen urls
+    #[getset(get = "pub")]
+    peer_listen_urls: Vec<String>,
+    /// Xline server peer advertise urls
+    #[getset(get = "pub")]
+    peer_advertise_urls: Vec<String>,
+    /// Xline server client listen urls
+    #[getset(get = "pub")]
+    client_listen_urls: Vec<String>,
+    /// Xline server client advertise urls
+    #[getset(get = "pub")]
+    client_advertise_urls: Vec<String>,
```

Comment:
What is the difference between `XX_listen_urls` and `XX_advertise_urls`?

Reply:
Same as etcd, the server's Tcp listener will bind to `XX_listen_urls`, and `XX_advertise_urls ` will be used to connect by other nodes or clients